src/state/wallet/reducer.ts

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from '@reduxjs/toolkit'
```

Comment:
Why does each wallet need its own error state?

Reply:
explained above, but the reason is it's easier to store it this way that to have a concept of a "pendingError" and "currentError"